src/main/webapp/search.js

```diff
@@ -110,3 +109,14 @@ function getRequestUrl() {
   url = URL_STRUCTURE + userSearchInput + '&key=' + YOUTUBE_API_KEY;
   return url;
 }
+
+/**
+ * Set apiKey variable to data read from file
+ */
+function retrieveApiKey() {
+  $(function() {
```

Comment:
Question: Do you need DOM to finish loading to apply this method?
https://stackoverflow.com/questions/7642442/what-does-function-do

Reply:
I believe so. That's why I have it call on window.onload so that the key is retrieved as soon as possible.